public/javascripts/controllers.js

```diff
@@ -736,6 +736,144 @@ angular.module('uforia')
 
       loadUsers();
 
+      $scope.archiveUserModal = function(user) {
+        $scope.editUser = user;
+        $scope.editUser.isDeleted = 0;
+        $scope.modalInstance = $modal.open({
+          templateUrl: 'views/modals/archiveUser.html',
+          size: 'md',
+          scope: $scope
+        });
+
+        $scope.modalInstance.opened.then(function () {
+
+          $scope.save = function(user) {
+            $scope.cuErrorMessages = [];
+
+            // Checks
+            if (typeof user.id === "undefined") {
+              $scope.cuErrorMessages.push('Id is required');
+            }
+
+            if ($scope.cuErrorMessages.length === 0) {
+              var addUser = true
+
+              // Save user
+              if (addUser) {
+                $http.post('/api/archive_user', $scope.editUser)
```

Comment:
Comment says `Save user` condition says `if(addUser)` and the api endpoint says `archive_user`. What are you doing?